glue/flagging_script_glue/flagging.py

```diff
@@ -889,8 +931,17 @@ def outlier_flag(df: pd.DataFrame) -> pd.DataFrame:
         df (pd.DataFrame): dataframe with 'is_outlier' column
     """
 
+    options = ["High price", "Low price", "High price (sqft)", "Low price (sqft)"]
+    pattern = r"\b(?:" + "|".join(map(re.escape, options)) + r")\b"
+
     df["sv_is_outlier"] = np.select(
-        [(df["sv_outlier_type"] == "Not outlier")], [0], default=1
+        [
+            df["sv_outlier_reason1"].str.contains(
+                pattern, case=False, na=False, regex=True
+            )
+        ],
+        [1],
+        default=0,
```

Comment:
Regex matching here to assign outlier status. This works because we need a price outlier for the observation to be classified as an outleir.